src/grb_constrs.jl

```diff
@@ -208,7 +208,7 @@ function get_constrs(model::Model, start::Integer, len::Integer)
     V = Array{Float64}(undef, nnz)
     for i in 1:length(cbeg)-1
         for j in (cbeg[i]+1):cbeg[i+1]
-            I[j] = i
+            I[j] = i + start - 1
```

Comment:
What is the implication of this? If anyone is using this, it will break existing code, correct? I'm tempted to allow it, because I don't think many (if any) people will depend on it, but maybe @mlubin has a different opinion.

Reply:
Yes, it would definitely mess up any dependencies.